### PR TITLE
stop creating secrets for dns, it uses the kube-system service account now

### PR DESCRIPTION
**Pull request by mikedanese**

### CODE REVIEW

**Comment by mikedanese** [comment]
cc @ArtfulCoder

**Comment by erictune** [comment]
LGTM.  Someday we will delete the old tokens from existing masters.

File: cluster/gce/configure-vm.sh [hunk 1]
```diff
@@ -585,7 +585,7 @@ function create-salt-master-auth() {
     # NB: If this list ever changes, this script actually has to
     # change to detect the existence of this file, kill any deleted
     # old tokens and add any new tokens (to handle the upgrade case).
-    local -r service_accounts=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring" "system:dns")
+    local -r service_accounts=("system:scheduler" "system:controller_manager" "system:logging" "system:monitoring")
```
Comment (erictune) [inline cluster/gce/configure-vm.sh]: Do you plan to delete the rest in a later PR?

**Comment by mikedanese** [comment]
@k8s-bot e2e test this please issue: #IGNORE

**Comment by mikedanese** [comment]
@k8s-bot e2e test this please issue: #IGNORE

**Pull request merged**